### PR TITLE
packaging: fix cloud-hypervisor build

### PR DESCRIPTION
**Pull request by bergwolf**
We should just download the official static build binary instead of
trying to build on our own.

Fixes: #760

Ref:
* failure: https://github.com/kata-containers/kata-containers/actions/runs/260953909
* success: https://github.com/bergwolf/kata-containers/runs/1134462133?check_suite_focus=true

### CODE REVIEW

**Review by fidencio — COMMENTED** [review]
@jcvenegas, what's your take on this? I thought building our own was porpusefully done to catch issues early.

OTOH, I totally understand using a released binary and avoiding breakages.

**Comment by bergwolf** [comment]
Per cloud-hypervisor maintainer's reply(https://github.com/cloud-hypervisor/cloud-hypervisor/issues/1723#issuecomment-694924725), we don't have guarantee that older clh releases can always build. It means that we cannot build it from source especially for stable releases. 
> At this point in time, due to the fast moving nature of the project, I don't think we can guarantee that old releases will continue to build.

**Review by fidencio — APPROVED** [review]
You have my ack but, please, wait for an answer from @jcvenegas before pushing.

**Review by amshinde — APPROVED** [review]

**Comment by amshinde** [comment]
@likebreath @jcvenegas PTAL as well. Seeing the issues that we saw recently in the release, I think we should go with this approach.

**Comment by likebreath** [comment]
It looks good to me that we are pulling static binaries from clh release directly.

There is a known bug for the released binaries of clh (https://github.com/cloud-hypervisor/cloud-hypervisor/issues/1669) where the version info is missing for the released binaries. This is the bug stops @jcvenegas and @egernst to consume the released binaries directly for kata releasing 1.x few weeks ago (see PR https://github.com/kata-containers/packaging/pull/1141 and its failing log [here](http://jenkins.katacontainers.io/job/kata-containers-packaging-ubuntu-18-04-PR/1180/console)).  

The above bug has been fixed in the latest clh v0.10.0 (was out earlier today). We will need to upgrade to clh v0.10.0 in kata first before we can leverage this change on packaging. 1) The upgrade should be quick. 2) But given our CLH CI in kata 2.0 is broken, I would prefer to fix the CI first today. I am looking at those two issues today, and will keep you posted.

Questions or advises if you have, please don't hesitate to share. Thanks.

**Comment by bergwolf** [comment]
Please let's do not block the release and fix clh CI and upgrade in the meantime. Thanks!

**Comment by likebreath** [comment]
For a reference, the CLH CI  for the tests repo has been fixed (issue https://github.com/kata-containers/tests/issues/2863).  The CLH CI in the kata-conatiners repo should also be fixed now (issue https://github.com/kata-containers/tests/issues/2858), but  we will need to confirm after landing the mentioned PR in the tests repo. 

Also, clh successfully is upgraded to v0.10.0 in kata 1.x (https://github.com/kata-containers/runtime/pull/2974). So I am not expecting complications to do the upgrade for kata 2.0. I will do it right after landing the PRs for fixing CLH CIs.

**Pull request merged**
